WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsAlerts.swift

```diff
@@ -1,4 +1,5 @@
 import UIKit
+import Hardware
```

Comment:
I would suggest importing Yosemite instead. Yosemite maps 
`public typealias CardReaderServiceError = Hardware.CardReaderServiceError
`

Reply:
Ah, I thought about adding the type alias, but I should have checked if it was already there.